src/renderer/containers/main/search.ts

```diff
@@ -56,11 +55,40 @@ class Search extends Container<SearchState, MainCTX> {
 
   }
 
+  _isTagMatch = ( note: NoteObj, filterTags: String[] ): boolean => {
+
+    const noteTags = this.ctx.note.getTags ( note );
+    if ( filterTags.length == 0 ) {
+      return true;
+    } else {
+      return noteTags.filter ( token => filterTags.indexOf ( token ) > -1 ) .length > 0;
+    }
+  };
```

Comment:
A small suggestion to make the code more readable and hopefully easier to maintain in the future.
 - added earlier return when `filterTags.lenght == 0` without invoking the `getTags`
 - using `includes` instead of `indexOf` for easier to reading.
 - changed the variable `token` to `noteTag` 
 - Looks like you have `lodash` import in the file I would consider using it for code convention.

```suggestion
  _isTagMatch = ( note: NoteObj, filterTags: String[] ): boolean => {
     if ( filterTags.length == 0 ) {
      return true;
    }
    
    const noteTags = this.ctx.note.getTags ( note );
    return noteTags.filter ( noteTag => filterTags.includes(noteTag) ) .length > 0;
  };
```